documentation/docs/payments.mdx

```diff
@@ -66,3 +66,43 @@ Content-Type: application/spsp4+json
   "shared_secret": "6jR5iNIVRvqeasJeCty6C+YB5X9FhSOUPCL/5nha5Vs="
 }
 ```
+
+## XRP Ledger
+
+The `application/xrpl+json` representation of an invoice or account resource
+includes the following **additional** properties for paying the receiver via the
+XRP Ledger:
+
+| Property       | Type               | Description                                                                              |
+| -------------- | ------------------ | ---------------------------------------------------------------------------------------- |
+| xrplAddress    | XRP Ledger Address | The XRP Ledger address to use when making payments to the account or against the invoice |
+| destinationTag | string             | The destination tag to use when making payments to the account or against the invoice    |
+
+The following is a non-normative example where the client gets the details to
+pay an account via the XRP Ledger:
+
+```http
+GET alice HTTP/1.1
+Host: wallet.example
+Accept: application/xrpl+json
```

Comment:
`application/xrpl+json` is not one of the standard/registered media-types. Payment networks could have different environments such as mainnet, testnet, devnet, etc. It is important to indicate the same in the client's `'Accept' `header as well as indicated in the response in some form (see the comment below for reasons). I would suggest to refer to PayID protocol RFC for more details on media-types.

`application/xrpl-testnet+json`

Reply:
> application/xrpl+json is not one of the standard/registered media-types

No, but this is a different protocol. The media type we use for ILP is also different to what is proposed by PayID.

> Payment networks could have different environments such as mainnet, testnet, devnet, etc. It is important to indicate the same in the client's 'Accept' header as well as indicated in the response in some form (see the comment below for reasons).

My assertion is that this is an unnecessary over complication so for Open Payments I'm proposing that we exclude this.

A simpler way to do this is to only issue identifiers in test domains if they are for test networks: e.g. `alice$testnet.rafiki.money`

Or, if absolutely necessary, leave the live network unchanged and add a suffix to the media-type for the test network to disambiguate them.

> I would suggest to refer to PayID protocol RFC for more details on media-types.

I have reviewed the RFC but this is a different protocol. 

Open Payments is only dependent on the PayID URI RFC. This PR proposes a new discovery protocols and uses Open Payments for setup (as opposed to PayID Protocol).

Reply:
> I have reviewed the RFC but this is a different protocol.

Yeah -- @aanchal4  and @adrianhopebailie, I think this is the disconnect between you. This proposal is re-defining some of what PayID can do by pulling it into the OpenPayments layer (specifically: where does one get the crypto address for a PayID). 

FWIW @adrianhopebailie (see my other comment above around where clients get address info), we've been doing a lot of thinking around this and have concluded that it makes sense to let the PayID server treat sometihng like an XRPL address on-par with a Payment Pointer -- in layman's terms, the PayID server gives you an address for the  payment-network you ask it for.

In that way, PayID can yield a Payment Network Address that can be brought to the OpenPayments server, and OP works roughly the same way for each -- though "[getting payment details](https://docs.openpayments.dev/payments)" becomes an optional step.

For example:

* When you bring an ILP-network address to OP, you need to do SPSP to get connection details. 
* In Crypto, you might want to get info about your non-custodial wallet, or do nothing. 
* In ACH you do nothing.

Reply:
> PayID server treat sometihng like an XRPL address on-par with a Payment Pointer -- in layman's terms, the PayID server gives you an address for the payment-network you ask it for.

That's not really how Open Payments works though. A Payment Pointer is not enough info to use to send money to someone. It's an identifier that you use to discover that address (the ILP Address).

None of the primitives in Open Payments are payment network specific, the only assumption we have is that if you start with a Payment Pointer then you're guaranteed to be able to get an ILP address for that account.

Payment Pointers and PayIDs are interchangeable, they only differ in that a PP identifies an account that is guaranteed to be accessible via ILP and a PayID identifies an account owner that may have multiple accounts on various networks. Both are used as identifiers to follow a discovery flow and ultimately resolve an account and payment network address(es) for that account.